access/msteams/plugindata.go

```
@@ -47,7 +56,12 @@ func EncodePluginData(data PluginData) map[string]string {
 
 	var encodedMessages []string
 	for _, msg := range data.TeamsData {
-		encodedMessages = append(encodedMessages, fmt.Sprintf("%s/%s/%s", msg.ID, msg.Timestamp, msg.RecipientID))
+		jsonMessage, err := json.Marshal(msg)
+		if err != nil {
+			continue
+		}
```

We discard this error
Can we at least add a log?